### PR TITLE
[CosmosDB] Regenerating sdk and renamed cosmos-db to cosmosdb

### PR DESCRIPTION
**Pull request by MehaKaushik**
temporary fix for AdditionalProperties failure

Updated tests and deleted extra SdkInfor file

Temp Fix for failing Long Running Operations

Updating the version and release notes

Rephrasing the release notes

minor addition in release notes

### CODE REVIEW

**Comment by MehaKaushik** [comment]
@isra-fel Can you please re-run the CI

**Comment by MehaKaushik** [comment]
@erich-wang @isra-fel Please review/merge this at your earliest convenience.

**Comment by MehaKaushik** [comment]
CI is successful, but on github UI it shows as in progress: 
image

**Comment by MehaKaushik** [comment]
Ping @isra-fel @erich-wang

**Comment by isra-fel** [comment]
Yes I confirmed the checks all passed in ADO

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @MehaKaushik sorry for the late response. I am just back to work from the holiday.
A few suggestions:
- Please edit your generate.ps1 to replace autorest version "latest" with "V2" and run it to generate again. This is because we found out the latest version didn't work well with C# generator.
- Please remove the redundant file src/Generated/code-model-v1 . This was also do to the latest autorest
- Please make sure you check in the metadata txt file under /eng/mgmt/mgmtmetadata , which is generated when you run `generate.ps1`
- I saw some models were removed, e.g. `AzureEntityResource` . Was that intentional? Cause if it is, then it's a breaking change, and we'll need to bump the major version, i.e. to 2.0.0 (as  well as `AssemblyVersion` in `AssemblyInfo.cs`

Thanks

**Comment by MehaKaushik** [comment]
Hey @isra-fel when I call the generate script with v2, the autorest generation takes forever. This is stuck for more than 15 minutes now. I have tried on two different machines, but the same outcome.

Please suggest an alternative solution.

image

**Comment by MehaKaushik** [comment]
> Hey @isra-fel when I call the generate script with v2, the autorest generation takes forever. This is stuck for more than 15 minutes now. I have tried on two different machines, but the same outcome.
> 
> Please suggest an alternative solution.
> 
> image

It generates the code, but after 15-20 minutes. Was able to eventually do it.

File: eng/Packages.Data.props [hunk 1]
```diff
@@ -31,6 +31,7 @@
     <PackageReference Update="Microsoft.Azure.KeyVault.Core" Version="3.0.3" />
     <PackageReference Update="Microsoft.Azure.Management.Batch" Version="4.2.0" />
     <PackageReference Update="Microsoft.Azure.Management.ContainerRegistry" Version="2.0.0" />
+    <PackageReference Update="Microsoft.Azure.Management.CosmosDB" Version="1.1.0" />
```
Comment (isra-fel) [inline eng/Packages.Data.props:34]: I'm not familiar with this props file. Could you explain why cosmosdb was added here?

Comment (isra-fel) [reply eng/Packages.Data.props:34]: Talked with Meha offline. This file was for maintaining dependencies in the repo. CosmosDB's manamagement plane SDK isn't for now used in mulptiple projects, so it's not necessary.

**Review by isra-fel — APPROVED** [review]

**Comment by MehaKaushik** [comment]
@isra-fel please re-run the CI

**Pull request merged**